TemporaryOysters/bangazon-cli
# Me product fixtures

##Pull Requests



##Descriptions:

* Please describe what issue this PR solves
Created a fixtures file containing 9 products to be used.










##Steps to Test:


Should work when product model has been created and tests pass. Import from command line.

##Related Issue:



##System configuration.


* Are there 3rd party libraries that need to be installed?
None

* Any command line utilities to run?
None
